.github/workflows/pr.yml

```diff
@@ -36,3 +36,41 @@ jobs:
 
       - name: Build docs
         run: dotnet fsdocs build --noapidocs --projects "$(pwd)/Fable.Daemon/Fable.Daemon.fsproj"
+
+      - name: Cache package
+        uses: actions/cache@v3
+        with:
+          path: |
+            *.tgz
+          key: ${{ runner.os }}-package-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-package-
+
+  
+  publish:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            *.tgz
+          key: ${{ runner.os }}-package-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-package-
+
+      - uses: oven-sh/setup-bun@v1
+
+      - name: Publish to npm
```

Comment:
You are publish on PR builds here I think.
Why?

Reply:
was trying to add a publish with tag so we can test from PRs externally, if not useful we can add this only in master?